site/class-1-bar/NataliaHartgers/readme.md

```diff
@@ -0,0 +1 @@
+# Hello World
```

Comment:
This branch is called bar, but there’s no bar chart here?

Also: your bar chart is late and I unfortunately don’t have time to review student work passed the due date.

You may add it though, and write a corresponding readme for the Readme?! assignment of class 2, and you can still receive points for the readme!